docs/source/starthere/fundamentals.rst

```diff
@@ -0,0 +1,240 @@
+NeMo fundamentals
+=================
+
+On this page we will go a little bit deeper on how NeMo works, to give you a good foundation for using NeMo for your :ref:`desired usecase <where_next>`.
+
+NeMo Models
+-----------
+
+A NeMo "model" includes all of the below components wrapped into a singular, cohesive unit:
```

Comment:
Suggested revision.

NVIDIA NeMo is a powerful framework for building and deploying neural network models, including those used in generative AI, speech recognition, and natural language processing. NeMo stands for “Neural Modules,” which are the building blocks of the models created using this platform. NeMo includes all of the following components wrapped into a singular, cohesive unit: